src/Configuration/Parser/BaseParser.php

```diff
@@ -21,22 +21,37 @@ abstract class BaseParser
     /** @var string[] */
     protected $filenames = [];
 
-    public function __construct()
+    /** @var string */
+    protected $filename;
```

Comment:
what is a difference here between $filename and $filenames?

Reply:
Done, see latest commit.